astropy/timeseries/sampled.py

```diff
@@ -141,7 +141,7 @@ def time(self):
         """
         return self['time']
 
-    def fold(self, period=None, midpoint_epoch=None):
+    def fold(self, period=None, epoch=None, normalize_phase=True, phase_range=1):
```

Comment:
I suggest keeping this argument and also having ``start_epoch`` and then users can choose which to use (and only one can be given). Having just ``epoch`` is not specific enough.

Reply:
locally I have now a version with `epoch_time` and `epoch_phase`, that would I think cover both use cases without the need of having both `midpoint_epoch` and `start_epoch`.

Reply:
Ok that works too. Could you do a deprecation of ``midpoint_epoch``? I know we said it was experimental but might as well give people some guidance on updating anyway?

Reply:
ok.